diff --git a/node_modules/@stoplight/json-schema-ref-parser/lib/pointer.js b/node_modules/@stoplight/json-schema-ref-parser/lib/pointer.js
index 3bcbb33..3d1ff16 100644
--- a/node_modules/@stoplight/json-schema-ref-parser/lib/pointer.js
+++ b/node_modules/@stoplight/json-schema-ref-parser/lib/pointer.js
@@ -95,12 +95,12 @@ Pointer.prototype.resolve = function (obj, options, pathFromRoot) {
     }
 
     let token = tokens[i];
-    if (this.value[token] === undefined || this.value[token] === null) {
+    if (this.value && (this.value[token] === undefined || this.value[token] === null)) {
       this.value = null;
-      throw new MissingPointerError(token, this.originalPath, this.path, pathFromRoot);
+      //throw new MissingPointerError(token, this.originalPath, this.path, pathFromRoot);
     }
     else {
-      this.value = this.value[token];
+      this.value = this.value?.[token] ?? null;
     }
   }
 
